app/uk/gov/hmrc/helptosavefrontend/controllers/HelpToSaveFrontendController.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosavefrontend.controllers
+
+import play.api.mvc.{Request, Result}
+import uk.gov.hmrc.helptosavefrontend.config.FrontendGlobal
+import uk.gov.hmrc.play.frontend.controller.FrontendController
+
+trait HelpToSaveFrontendController extends FrontendController {
```

Comment:
we can make this class extend `Logging`  and `HelpToSaveAuth` - but in a future PR
LGTM